src/styles/components/card.scss

```diff
@@ -19,7 +19,7 @@ $block: ns(card);
     $tn-card-media-width: 75px !default;
     $tn-card-color-border: tn-color(neutral,2) !default;
     $tn-card-color-disabled: tn-color(text,3) !default;
-    $tn-card-background-color: tn-color(inverse) !default;
+    $tn-card-background-color: tn-color(text-inverse) !default;
```

Comment:
@joseegm This is more of a question for @LeoT7508. The `neutral-1` shade is not white. Do we need a backgrounds color palette or add 'white' to neutrals (which isn't perfectly logical)? 

Another option is to change `text` to something like `base` or `default` where it's more clear that `inverse` is meant to be the inverse of the base palette.